stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/channel/add/AddChannelMembersAdapter.kt

```diff
@@ -0,0 +1,43 @@
+package io.getstream.chat.android.ui.channel.add
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import io.getstream.chat.android.client.models.User
+import io.getstream.chat.android.client.models.name
+import io.getstream.chat.android.ui.databinding.StreamAddChannelMemberItemBinding
+
+internal class AddChannelMembersAdapter :
+    ListAdapter<User, AddChannelMembersAdapter.MemberViewHolder>(
+        object : DiffUtil.ItemCallback<User>() {
+            override fun areItemsTheSame(oldItem: User, newItem: User): Boolean {
+                return oldItem.id == newItem.id
+            }
+
+            override fun areContentsTheSame(oldItem: User, newItem: User): Boolean {
+                return oldItem == newItem
+            }
+        }
+    ) {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): MemberViewHolder {
+        return StreamAddChannelMemberItemBinding
+            .inflate(LayoutInflater.from(parent.context), parent, false)
+            .let { MemberViewHolder(it) }
```

Comment:
Nit: it could be also written in the following way:
```kotlin
return StreamAddChannelMemberItemBinding
            .inflate(LayoutInflater.from(parent.context), parent, false)
            .let(::MemberViewHolder)
```

Reply:
Nit nit: the comment should've been a change suggestion 😄